File: jinja2/parser_test.go
package jinja2
import (
	"testing"
	"fmt"
)

func TestParser(t *testing.T) {
	parser := NewParser()
	input := parser.Parse("{{ foo }}")
	fmt.Println(input)
	// ast := parser.Parse("{{ foo }}")
	// if ast == nil {
	// 	t.Fatal("parser.Parse returned nil")
	// }
	// if ast.Type != "Root" {
	// 	t.Fatalf("expected Root node, got %s", ast.Type)
	// }
}

File: jinja2/parser_test.py
import parser
# import pytest
# import jinja2.parser

# jinja2.parse("dddd")

def test_jinja2():
    print("------->test_jinja2")
    inputs=parser.parse("dddd")
    print(inputs)
    assert 1 == 1

if __name__ == '__main__':
    # pytest.main(["-v", "-s", "jinja2_test.py"])
    test_jinja2()